TripKitUI/core/TransportKitUI/SGTripSegmentsView.h

```diff
@@ -27,6 +27,19 @@
  */
 @property (nonatomic, assign) BOOL colorCodingTransitIcon;
 
+/**
+ This color is used for darker texts. In addition, this is also the color which
+ will be used to tint the transport mode icons if `colorCodingTransitIcon` is
+ set to NO.
+ */
+@property (nonatomic, strong) UIColor *darkGreyColor;
```

Comment:
Please rename these properties to `darkTextColor` and `lightTextColor`.

Reply:
Also please make `nullable` and say what the default colour will be if it's set to `nil`.

Same comments apply to all the other colour properties that you added.

Reply:
Done